kxml_compiler/tests/parserxsdtest.cpp

```diff
@@ -29,7 +29,7 @@
 void ParserXsdTest::initTestCase()
 {
     RNG::ParserXsd parser;
-    QFile file("simple.xsd");
+    QFile file(QFINDTESTDATA("simple.xsd"));
```

Comment:
It's good practice to verify that QFINDTESTDATA didn't return an empty string.

Reply:
Added `QVERIFY` check